src/markdown/tutorial/part-2/11-ember-data.md

```diff
@@ -228,15 +227,18 @@ Alright, now that we have our model set up, it's time to refactor our route hand
 -
 -    return { id, type, ...attributes };
 +  async model(params) {
-+    return this.store.findRecord('rental', params.rental_id);
++    const { content } = await this.store.request(
```

Comment:
having `content.data` instead of just `data` as in Axios for example, increase complexity :/

Reply:
it does not, but we also aren't revisiting this design at this point.

Reply:
we might do some `@warp-drive/ember` utils for better DX ?